snuba/query/query.py

```diff
@@ -129,31 +127,37 @@ def get_all_expressions(self) -> Iterable[Expression]:
             chain.from_iterable(map(lambda orderby: orderby.node, self.__order_by)),
         )
 
-    def transform_expressions(
-        self,
-        func: Callable[[Expression], Expression],
-    ) -> None:
+    def transform_expressions(self, func: Callable[[Expression], Expression],) -> None:
         """
         Transforms in place the current query object by applying a transformation
         function to all expressions contained in this query
+
+        Cointrarily to Expression.transform this happens in place since Query has
```

Comment:
```suggestion
        Contrary to Expression.transform this happens in place since Query has
```

(I suppose this is already on master, though.)